litmus/VULKAN/Manual/counter-atomic-load.litmus

```diff
@@ -0,0 +1,13 @@
+Vulkan counter-load-atomic

```

Comment:
I think the following naming scheme is more descriptive for these 4 variants of the test
- counter-plain-store-atomic-load
- counter-atomic-store-rmw
- counter-plain-store-plain-load
- counter-plain-store-rmw

Reply:
ok